src/Mappers/Root/NullableTypeMapperAdapter.php

```diff
@@ -126,7 +126,7 @@ private function getNonNullable(Type $type): Type|null
             return null;
         }
         if ($type instanceof Nullable) {
-            return $type->getActualType();
+            return $this->getNonNullable($type->getActualType());
```

Comment:
Does this read weird, or am I missing something here?

If $type is an instance of nullable, get non-nullable?